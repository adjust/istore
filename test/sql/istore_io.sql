BEGIN;
CREATE TABLE istore_io AS SELECT '1=>1,1=>2'::istore;
SELECT * FROM istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE istore_io AS SELECT '"1"=>"1","1"=>"2"'::istore;
SELECT * FROM istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE istore_io AS SELECT '"1"=>"1","-1"=>"2"'::istore;
SELECT * FROM istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE istore_io AS SELECT '"-1"=>"+1","1"=>"2"'::istore;
SELECT * FROM istore_io;
ROLLBACK;
