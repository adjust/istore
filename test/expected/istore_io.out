BEGIN;
CREATE TABLE istore_io AS SELECT '1=>1,1=>2'::istore;
SELECT * FROM istore_io;
  istore  
----------
 "1"=>"3"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE istore_io AS SELECT '"1"=>"1","1"=>"2"'::istore;
SELECT * FROM istore_io;
  istore  
----------
 "1"=>"3"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE istore_io AS SELECT '"1"=>"1","-1"=>"2"'::istore;
SELECT * FROM istore_io;
       istore       
--------------------
 "-1"=>"2","1"=>"1"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE istore_io AS SELECT '"-1"=>"+1","1"=>"2"'::istore;
SELECT * FROM istore_io;
       istore       
--------------------
 "-1"=>"1","1"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE istore_io AS SELECT ' "-1"=>"+1","1"=>"2"'::istore;
SELECT * FROM istore_io;
       istore       
--------------------
 "-1"=>"1","1"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE istore_io AS SELECT '"-1"=> "+1","1"=>"2"'::istore;
SELECT * FROM istore_io;
       istore       
--------------------
 "-1"=>"1","1"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE istore_io AS SELECT '"-1"=>"+1" ,"1"=>"2"'::istore;
SELECT * FROM istore_io;
       istore       
--------------------
 "-1"=>"1","1"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE istore_io AS SELECT '"-1"=>"+1", "1"=>"2"'::istore;
SELECT * FROM istore_io;
       istore       
--------------------
 "-1"=>"1","1"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE istore_io AS SELECT '"-1"=>"+1","1"=>"2" '::istore;
SELECT * FROM istore_io;
       istore       
--------------------
 "-1"=>"1","1"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE istore_io AS SELECT '-1=>+1,1=>2 '::istore;
SELECT * FROM istore_io;
       istore       
--------------------
 "-1"=>"1","1"=>"2"
(1 row)

ROLLBACK;
